nilearn/regions/signal_extraction.py

```diff
@@ -128,8 +245,8 @@ def img_to_signals_labels(imgs, labels_img, mask_img=None,
     # Set to zero signals for missing labels. Workaround for Scipy behaviour
     missing_labels = set(labels) - set(np.unique(labels_data))
     labels_index = dict([(l, n) for n, l in enumerate(labels)])
-    for l in missing_labels:
-        signals[:, labels_index[l]] = 0
+    for i in missing_labels:
+        signals[:, labels_index[i]] = 0
```

Comment:
I'm not sure to understand why you changed this.

Reply:
Just a flavor.